tmt/steps/execute/internal.py

```diff
@@ -221,53 +225,128 @@ def check(self, test):
 
     def _setup_reboot_script(self, guest):
         """ Set up reboot script on the guest. """
-        def setup_file(filename, content):
-            file_path = os.path.join(self.workdir, filename)
-            with open(file_path, 'w') as template:
-                template.write(content)
-            # Make it executable, keep perms when pushing
-            perms = os.stat(file_path)
-            os.chmod(file_path, perms.st_mode | stat.S_IEXEC)
-            return file_path
+        script = '/restraint/rstrnt-reboot'
```

Comment:
I'd suggest making this a constant at the top of the file (upper-case), same for the other script